inc/components/post-featured-media/component.php

```diff
@@ -29,41 +29,17 @@
 			// Get the post ID from a context provider.
 			$post_id = $component->get_config( 'post_id' );
 
-			// Get and validate image url.
-			$image_url = get_the_post_thumbnail_url( $post_id );
-			if ( empty( $image_url ) ) {
-				return $component;
-			}
-
-			$wrapper = new Component( 'irving/container' );
-			$wrapper->set_config( 'max_width', 'lg' );
-			$wrapper->set_config( 'style', [ 'bottom-margin' => '1rem' ] );
-
-			$wrapper->append_child(
-				( new Component( 'material/card-content' ) )
-					->set_config( 'gutter_bottom', true )
-					->set_child(
-						( new Component( 'material/card-media' ) )
-							->set_config( 'image', $image_url )
-							->set_config(
-								'style',
-								[
-									'bottom_margin' => '1rem',
-									'height'        => '450px',
-								]
-							)
-					)
+			$component->append_child(
+				[
+					'name'   => 'irving/post-featured-image',
+					'config' => [
+						'aspect_ratio' => $component->get_config( 'aspect_ratio' ),
+						'object_fit'   => $component->get_config( 'object_fit' ),
+					],
```

Comment:
This is temporary syntax. I want to figure out a better way of doing this in one action.